projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/KubernetesService.java

```diff
@@ -198,6 +199,7 @@ private enum ContainerResourceType {
    */
   protected KubernetesService(
       String namespace, String kubeconfig, boolean k8sSupportsV1CronJob, Logger log) {
+    assertTrue(StringUtils.isNotBlank(namespace));
```

Comment:
Maybe this could print an adequate error message is it is blank?

Reply:
Done.